### PR TITLE
French Minor

### PR DESCRIPTION
**Pull request by hannahchoi05**
**References**
- Linear: https://linear.app/hoagie/issue/DEV-68/french-and-italian-minor

**Proposed Changes**
- Added [French minor](https://ua.princeton.edu/fields-study/departmental-majors-degree-bachelor-arts/french-and-italian)

### CODE REVIEW

File: backend/minors/FRE.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    by the fall of junior year, but students are strongly encouraged to do so by the spring of their
+    sophomore year. Applications to the minor program are accepted until the end of spring of junior year.
+  completed_by_semester: 5
+  course_list: 
+  - FRE 107
+  - FRE 108
+  - FRE 1027
+- name: Program of Study
+  max_counted: 1
+  min_needed: 5
```
Comment (ggchiriac) [inline backend/minors/FRE.yaml:35]: Some errors here

**Review by ggchiriac — CHANGES_REQUESTED** [review]
Some errors with max_counted, min_needed.

File: backend/minors/FRE.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    A 200-level course is a prerequisite for taking 300-level courses in French.
+  req_list: 
+  - name: Advanced Level
+    max_counted: 1
+    min_needed: 3
+    course_list: 
+    - FRE 3*
+  - name: Intermediate Level
+    max_counted: 1
+    min_needed: 0
```
Comment (issacli-0821) [inline backend/minors/FRE.yaml:50]: `max_counted` should be 2 and `min_needed` can just be 1

File: backend/minors/FRE.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    course_list: 
+    - FRE 2*
+    - name: Language Double Counting
+      req_list:
+        min_counted: 0
```
Comment (issacli-0821) [inline backend/minors/FRE.yaml:55]: We can set this to 1 too, generally we don't want these fields to be 0

Comment (issacli-0821) [inline backend/minors/FRE.yaml:55]: We can set this to 1 too, generally we don't want these fields to be 0

File: backend/minors/FRE.yaml (hunk 2 above)
Comment (issacli-0821) [inline backend/minors/FRE.yaml:50]: `max_counted` should be 2 and `min_needed` can just be 1

File: backend/minors/FRE.yaml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  min_needed: 5
+  explanation: |- 
+    Students must successfully take a minimum of five courses in the relevant language, linguistics, 
+    literature or culture, excluding courses that do not have a language prerequisite and any 
+    100-level language course. At least three of these courses must be at the 300 level (or higher). 
+    Up to two of these courses may be taken at the 200 level, but FRE 207 and 208 cannot both be counted. 
+    A 200-level course is a prerequisite for taking 300-level courses in French.
+  req_list: 
+  - name: Advanced Level
+    max_counted: 1
```
Comment (issacli-0821) [inline backend/minors/FRE.yaml:44]: This should be 5

**Review by issacli-0821 — CHANGES_REQUESTED** [review]
Left some comments. Let me know if you have any questions

File: backend/minors/FRE.yaml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  - name: Advanced Level
+    max_counted: 5
+    min_needed: 3
+    course_list: 
+    - FRE 3*
+  - name: Intermediate Level
+    max_counted: 2
+    min_needed: 1
+    course_list: 
+    - FRE 2*
```
Comment (issacli-0821) [inline backend/minors/FRE.yaml:52]: We can't have both a `course_list` and a `req_list`. Let's put all 2** courses in a requirement, and then put 207 and 208 in the `excluded_work_list`

**Review by issacli-0821 — CHANGES_REQUESTED** [review]
Left one comment and then we should be good!

**Review by issacli-0821 — APPROVED** [review]
Looks good!

**Pull request merged**